update_configs.py

```diff
@@ -0,0 +1,25 @@
+"""
+Update configurations for VSCode and other editors.
+VSCode at least does not support composing configurations (https://github.com/microsoft/vscode/issues/15909)
+and we don't want to clobber any existing configurations.
+"""
+import json
+from pathlib import Path
+from typing import Any, Dict
+
+VSCODE_CONFIG_DIR = Path('.vscode')
```

Comment:
Would it make sense to be more explicit about the path here:

```suggestion
VSCODE_CONFIG_DIR = Path(__file__).parent / '.vscode'
```
Not sure if this could get executed with a different working dir for some reason and end up putting files in weird places